doc/api/async_hooks.md

```diff
@@ -973,6 +973,21 @@ chains. It allows storing data throughout the lifetime of a web request
 or any other asynchronous duration. It is similar to thread-local storage
 in other languages.
 
+## Difference with raw Async Hooks
```

Comment:
It's better to avoid using new sections here and in below, as further content is dedicated to `AsyncLocalStorage` class docs.